### PR TITLE
Feature/sublist player registration

### PR DESCRIPTION
**Pull request by bobbyesh**
Implemented player sublist registration and un-registration.  No changes to the data model were made.

### CODE REVIEW

**Comment by bobbyesh** [comment]
I actually found that the sublist table was not needed.  If players register for a session within a division, is this sufficient?

**Comment by bobbyesh** [comment]
One thing not implemented is date picking.  When a player registers, the current time is saved.

**Comment by bobbyesh** [comment]
Implemented date picking too.

File: SnookR/main/templates/main/base.html [hunk 1]
```diff
@@ -30,6 +30,7 @@
           <!-- Collect the nav links, forms, and other content for toggling -->
           <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
             <ul class="nav navbar-nav navbar-right">
+              <li><a href="{% url 'division' %}">Divisions</a></li>
```
Comment (eSmelser) [inline SnookR/main/templates/main/base.html]: This pukes if you are not logged in OR if you are logged in as any user without a player attached (such as superuser), so we should either disable this when not logged in, handle the error it throws in a meaningful way, or both

**Review by eSmelser — APPROVED** [review]
Bugfix for only known issue. PR ready to go.

**Pull request merged**